package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "start": "webpack serve --open",
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d dist"
```

Comment:
@tkz96 there is no need to push the dist folder while you deployed your project using gh-pages. to remove the complexity in files for your repo.

Reply:
I am pushing the `dist` folder so that only the files inside the `dist` folder are visible to anyone viewing source. This is to practice security.